hakaiApi/DESCRIPTION

```diff
@@ -19,6 +19,7 @@ Depends:
 Imports:
     dplyr,
     httr,
+    httr2,
```

Comment:
Is `httr` still required with the addition of `httr2`? Can we get ride of `httr`?

Reply:
httr is not required.

Reply:
and yes I was also looking at how to publish it. something about uploading a tar.gz  via a web form?